modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/call/metric/ItEnabledMetricCallsTest.java

```diff
@@ -69,15 +75,8 @@ void nodeMetricSourcesList() {
         // Then
         assertThat(output.hasError()).isFalse();
 
-        MetricSource[] expectedMetricSources = {
-                new MetricSource().name("jvm").enabled(true),
-                new MetricSource().name("client.handler").enabled(false),
-                new MetricSource().name("sql.client").enabled(false),
-                new MetricSource().name("sql.plan.cache").enabled(false)
-        };
-
         // And
-        assertThat(output.body()).contains(expectedMetricSources);
+        assertThat(output.body()).contains(new MetricSource().name("jvm").enabled(true));
```

Comment:
didn't catch the change. We should check that we have the expected list of metrics, but checks only 'jvm' one

Reply:
Test method description states "Should display enabled jvm node metric source when cluster is up and running" :thinking:

Reply:
I think I got your point, I can leave these code unchanged :blush: 

Reverted this change, thanks